pkgs/tools/misc/lorri/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, pkgs
+, fetchFromGitHub
+, rustPlatform
+  # Updater script
+, runtimeShell
+, writeScript
+  # Apple dependencies
+, CoreServices
+, Security
+, cf-private
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lorri";
+  version = "rolling-release-2019-10-30";
+
+  meta = with stdenv.lib; {
+    description = "Your project's nix-env";
+    homepage = "https://github.com/target/lorri";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ grahamc Profpatsch ];
+  };
+
+  src = fetchFromGitHub {
+    owner = "target";
+    repo = pname;
+    # Run `eval $(nix-build -A lorri.updateScript)` after updating the revision!
+    # See passthru.update below for what this does.
+    rev = "03f10395943449b1fc5026d3386ab8c94c520ee3";
+    sha256 = "0fcl79ndaziwd8d74mk1lsijz34p2inn64b4b4am3wsyk184brzq";
+  };
+
+  passthru.updateScript = with builtins; writeScript "copy-runtime-nix.sh" ''
+    #!${runtimeShell}
+    cp ${src}/nix/runtime.nix ${toString ./runtime.nix}
+    cp ${src}/nix/runtime-closure.nix.template ${toString ./runtime-closure.nix.template}
+  '';
```

Comment:
please add a `passthru.tests.lorri = nixosTests.lorri` here too, so people can build `lorri.tests` (see the bazel and gitlab packages for other examples).

Reply:
Done, thanks!